features/htcondor/client/ce.pan

```diff
@@ -0,0 +1,50 @@
+unique template features/htcondor/client/ce;
+
+variable CONDOR_CONFIG={
```

Comment:
You should set the variable like this: variable CONDOR_CONFIG ?= {
as it can be useful to set it manually (i.e. for development purpose)